### PR TITLE
update to generator-jhipster 8.3.0

### PR DESCRIPTION
**Pull request by mshima**
- move the micronaut generator logic and templates for server to micronaut.
- use java/code-quality, java/node, java/jib generators.

### CODE REVIEW

File: generators/micronaut/generator.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+  }
+
+  get [BaseApplicationGenerator.PREPARING]() {
+    return this.asPreparingTaskGroup({
+      loadDependencies({ application }) {
+        application.micronautDependencies = { ...mnConstants.versions };
+        this.loadJavaDependenciesFromGradleCatalog(application.micronautDependencies);
+        Object.assign(application.javaDependencies, application.micronautDependencies);
+
+        // Workaound liquibase generator bug
```
Comment (mraible) [inline generators/micronaut/generator.js:88]:
```suggestion
        // Workaround liquibase generator bug
```
Is there a link to this issue so we'll know if we can remove it in the future?

File: generators/constants.cjs [hunk 2]
```diff
@@ -64,8 +64,6 @@ module.exports = {
     javassist: '3.27.0-GA', // Should match Hibernate deps
     javaxMail: '2.0.1',
     jbcrypt: '0.4',
-    liquibase: '4.27.0',
-    liquibaseHibernate6: '4.27.0',
     logback: '1.5.3',
```
Comment (mraible) [inline generators/constants.cjs:67]: Micronaut 4.3.8 is the latest version, but we can update to that in another PR.

**Review by mraible — APPROVED** [review]

File: generators/constants.cjs (hunk 2 above)
Comment (mraible) [reply generators/constants.cjs:67]: Created PR at https://github.com/jhipster/generator-jhipster-micronaut/pull/335.

File: generators/micronaut/generator.js (hunk 1 above)
Comment (mshima) [reply generators/micronaut/generator.js:88]: Should be fixed in main already.

Comment (mshima) [reply generators/micronaut/generator.js:88]:
```suggestion
        // Workaround liquibase generator bug. Drop for generator-jhipster 8.3.1.
```

**Pull request merged**